### DISCUSSION TITLE
Best approach for supporting 11 actuators distributes in 4 zones

### DISCUSSION THREAD

**Discussion by brainstormi** [nliaudat/esp32_8ch_motor_shield#53] [General]
Hi...
    I'm trying to setup it to support a 11 actuator installation than control 4 different zones (rooms where I have a thermostast). This implies that 1 room can has more than 1 actuator associated to it to cover all the space.
    Would it be feasible to connect more than 1 actuactor to a single channel in board? 
    If not and I need to expand the number of channels, in the readme it says that 1 board could be extended to 16 channels extending shift registers, but there no additional info/diagrams about how to do this. Could you elaborate a bit more how to extend the number of channels?.
     I saw than the recommendation would be to use 2 boards for this purpose, I was trying to minimize the number of devices installed to simplify the maintenance overtime if feasible.
Regards,

**Comment by nliaudat** [comment]
Hi, 
You cannot use 2 actuators on the same channel cause the endstops are differents for each actuator. 

You can chain 2 boards like explained in https://github.com/nliaudat/esp32_8ch_motor_shield/blob/main/shiftregOut.md
